test/suite/test_tiered04.py

```diff
@@ -153,9 +153,11 @@ def test_tiered(self):
         time.sleep(1)
         self.pr("Check removal of ")
         self.pr(self.obj1file)
-        self.assertFalse(os.path.exists(self.obj1file))
-        remove2 = self.get_stat(stat.conn.local_objects_removed, None)
-        self.assertTrue(remove2 > remove1)
+        # FIXME-WT-10838: We can't remove files from open tables because we don't know whether 
+        # there are active read reqeusts to those files.
```

Comment:
Typo `requests`

Reply:
Fixed. I'm also going to open a ticket to have `s_string` spell check comments from python files.